molecule/cookiecutter/scenario/driver/docker/{{cookiecutter.molecule_directory}}/{{cookiecutter.scenario_name}}/Dockerfile

```diff
@@ -1,3 +1,3 @@
 FROM centos:7
 
-RUN yum makecache fast && yum update -y && yum install -y python sudo yum-plugin-ovl && sed -i 's/plugins=0/plugins=1/g' /etc/yum.conf
+RUN yum makecache fast && yum update -y && yum install -y sudo yum-plugin-ovl && sed -i 's/^plugins=0$/plugins=1/g' /etc/yum.conf && yum clean all
```

Comment:
- python is already part of the base image
- add more specificity to the sed replace to enable yum plugins
- add `yum clean all` to remove yum crud from resulting image.